src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -35,6 +35,12 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
+// @todo #93:30min NHD needs to be tested against the following after #103 is
```

Comment:
@llorllale As I understand from #103, the overall report will fail if one of the metrics returns ``NaN``, right? So, until 103 is fixed, wouldn't it be better if this metric returned ``0`` instead of ``NaN``? Then, when 103 is fixed we go back to ``NaN`` here and add those tests.

If so, please update the puzzle to reflect also that.

Reply:
@amihaiemil from #103, it's the way tests are structured in `MetricsTest` that is not compatible with reports that may return `NaN` as a value. But for the regular jpeek use, the report itself does not "fail". In other words, the report generation does not throw errors and the output html renders without issues in a browser. It *could* be that the mean, average, variance, and so on, are thrown off for NHD if any classes result in `NaN`, but having the implementation return `0` will probably have the same effect.

So in other words, I believe we gain nothing by the change you suggested. WDYT?

Reply:
@llorllale I see. Let's leave it like that then. I figured that if the tests wouldn't work, then the App wouldn't as well :)